File: APPManage/AM.Model/safe_lock.cs
/**  版本信息模板在安装目录下，可自行修改。
* safe_lock.cs
*
* 功 能： N/A
* 类 名： safe_lock
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2016/8/12 22:32:19   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace AM.Model
{
	/// <summary>
	/// safe_lock:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class safe_lock
	{
		public safe_lock()
		{}
		#region Model
		private string _guid;
		private string _userid;
		private DateTime? _createdate;
		private string _wtdw;
		private string _ypmc;
		private string _zzcs;
		private string _xhgg;
		private string _ccbh;
		private string _jyrq;
		private string _jyyj;
		private string _jyjl;
		/// <summary>
		/// 
		/// </summary>
		public string Guid
		{
			set{ _guid=value;}
			get{return _guid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string userid
		{
			set{ _userid=value;}
			get{return _userid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? CreateDate
		{
			set{ _createdate=value;}
			get{return _createdate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string WTDW
		{
			set{ _wtdw=value;}
			get{return _wtdw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string YPMC
		{
			set{ _ypmc=value;}
			get{return _ypmc;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string ZZCS
		{
			set{ _zzcs=value;}
			get{return _zzcs;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string XHGG
		{
			set{ _xhgg=value;}
			get{return _xhgg;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string CCBH
		{
			set{ _ccbh=value;}
			get{return _ccbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string JYRQ
		{
			set{ _jyrq=value;}
			get{return _jyrq;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string JYYJ
		{
			set{ _jyyj=value;}
			get{return _jyyj;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string JYJL
		{
			set{ _jyjl=value;}
			get{return _jyjl;}
		}
		#endregion Model

	}
}

File: AM/Scripts/appm/safe_lock.js
function test() {
    var isok = false;
    if ($("#wtcompany").val() == "") {
        alert("委托单位不能为空");
        $("#wtcompany").focus();
    } else {
        if ($("#ypname").val() == "") {
            alert("样品名称不能为空");
            $("#ypname").focus();
        } else {
            if ($("#scstore").val() == "") {
                alert("制造厂商不能为空");
                $("#scstore").focus();
            } else {
                if ($("#type").val() == "") {
                    alert("型号/规格不能为空");
                    $("#type").focus();
                } else {
                    if ($("#cccode").val() == "") {
                        alert("出厂编号不能为空");
                        $("#cccode").focus();
                    } else {
                        if ($("#jydate").val() == "") {
                            alert("接样日期不能为空");
                            $("#jydate").focus();
                        } else {
                            if ($("#checkyj").val() == "") {
                                alert("检验依据不能为空");
                                $("#checkyj").focus();
                            } else {
                                if ($("#checkend").val() == "") {
                                    alert("检验结论不能为空");
                                    $("#checkend").focus();
                                } else
                                    isok = true;
                            }
                        }
                    }
                }
            }
        }
    }
    return isok;
}



String.prototype.trim = function () {

    return this.replace(/(^\s*)|(\s*$)/g, '');
}

function submit() {
    var check = test();
    if (check)
    {
    var userid = $("#userid").val();
    var objArr = $("input[type=text]")
    var textareas = $("textarea");
    
    var json = {
        "safe_lock": [{
            "Guid": "",
            "CreateDate": "",
            "userid": userid,
            "WTDW": $(objArr[0]).val(),
            "YPMC": $(objArr[1]).val(),
            "ZZCS": $(objArr[2]).val(),
            "XHGG": $(objArr[3]).val(),
            "CCBH": $(objArr[4]).val(),
            "JYRQ": $(objArr[5]).val(),
            "JYYJ": $(textareas[0]).val(),
            "JYJL": $(textareas[1]).val()
        }]
    };
    $.ajax({
        type: "post",
        url: "/AppM/addsafe_lock",
        data: { obj: JSON.stringify(json) },
        success: function (data) {
            layer.msg("提交成功");
            //alert("哈哈" + data);
            //debugger;
            //data = eval('(' + data + ')');

        }
    })
    //$.ajax({
    //    type: "POST",     
    //    contentType: "application/json; charset=utf-8",  
    //    dataType: "json",   
    //    url: "/AppM/addbasket_train",   
    //    data: "RID:" + json,
    //    success: function (data) {
    //        alert(data.b);       
    //    },
    //    error: function (err) { //如果执行不成功，那么执行此方法
    //        alert("err:" + err);
    //    }
    //});
    }
}